### PR TITLE
Remove graceful query parameter from stop program APIs

### PR DESCRIPTION
**Pull request by snehalakshmisha**
Jira: https://cdap.atlassian.net/browse/CDAP-19315

### CODE REVIEW

File: cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java [hunk 1]
```diff
@@ -296,49 +296,18 @@ public void performRunLevelStop(HttpRequest request, HttpResponder responder,
                                   @PathParam("app-id") String appId,
                                   @PathParam("program-type") String type,
                                   @PathParam("program-id") String programId,
-                                  @PathParam("run-id") String runId,
-                                  @QueryParam("graceful") String gracefulShutdownSecs) throws Exception {
```
Comment (de-lan) [inline cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java:300]: Any specific reasons we're removing it now? I think we added it in UI not longtime ago?
https://github.com/cdapio/cdap-ui/pull/418

File: cdap-runtime-ext-dataproc/pom.xml [hunk 2]
```diff
@@ -29,11 +29,6 @@
   <name>CDAP Google Cloud Dataproc Runtime Extension</name>
   <packaging>jar</packaging>
 
-  <properties>
```
Comment (chtyim) [inline cdap-runtime-ext-dataproc/pom.xml]: Why all these changes??

File: cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java [hunk 3]
```diff
@@ -83,7 +83,7 @@ public static void deleteGCSPath(Storage storageClient, String bucket, String pa
       boolean addedToDelete = false;
       for (Blob blob : blobs.iterateAll()) {
         LOG.trace("Added path to be deleted {}", blob.getName());
-        batch.delete(blob.getBlobId(), Storage.BlobSourceOption.generationMatch());
```
Comment (chtyim) [inline cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java]: Seems like you brought in changes from develop branch? They shouldn't be here.

Comment (snehalakshmisha) [reply cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java]: Oh no, this commit gives an error with the way I am updating modules. So reverted it for testing. It is not part of the final PR

File: cdap-runtime-ext-dataproc/pom.xml (hunk 2 above)
Comment (snehalakshmisha) [reply cdap-runtime-ext-dataproc/pom.xml]: Not part of the PR

File: cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java (hunk 1 above)
Comment (snehalakshmisha) [reply cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java:300]: Discussed offline

**Review by albertshau — APPROVED** [review]
lgtm

**Pull request merged**